community.md

```diff
@@ -79,6 +79,18 @@ community and for maintaining a queue of development work and as the public
 record of work on the project. We use the mailing lists for development discussions,
 where a lengthy discussions is required.
 
+### Meetings
+
+We host online meetings to provide venues for synchronous discussions about the Arrow project. These discussions usually focus on topics of interest to developers who are contributing to Arrow, but we welcome users of Arrow to join. Currently there are two series of regularly scheduled meetings:
+
+<ul>
+  <li>Arrow community meting, held biweekly</li>
+  <li>Arrow R package development meeting, held biweekly</li>
+</ul>
```

Comment:
Isn't there also a rust meeting? @iajoiner ?

Reply:
Oh, it looks like there is, thanks for pointing this out @assignUser! I recall that the Rust sync meeting was cancelled for a while in early 2022 but [it looks like it has resumed](https://docs.google.com/document/d/1atCVnoff5SR4eM4Lwf2M1BBJTY6g3_HUNR6qswYJW_U/). I'll add this here.

@iajoiner if people are interested in attending the Arrow Rust sync meeting, where should they go to get more information?

Reply:
@assignUser @ianmcook Yes. Let me add the information. It is here: https://docs.google.com/document/d/1atCVnoff5SR4eM4Lwf2M1BBJTY6g3_HUNR6qswYJW_U/edit